frontend/src/scenes/billing/billingLogic.tsx

```diff
@@ -172,6 +172,12 @@ export const billingLogic = kea<billingLogicType>([
                 },
 
                 deactivateProduct: async (key: string) => {
+                    // clear upgrade params from URL
+                    const currentURL = new URL(window.location.href)
+                    currentURL.searchParams.delete('upgraded')
+                    currentURL.searchParams.delete('products')
```

Comment:
Is this necessary? The page will reload upon deactivateProduct success.

Reply:
Yeah, if those are there the page thinks an upgrade happened successfully, so we need to make sure they're removed before reload.